presto-main/src/main/java/com/facebook/presto/sql/gen/ExpressionCompiler.java

```diff
@@ -96,7 +96,7 @@ public PageProcessor compilePageProcessor(RowExpression filter, List<RowExpressi
         try {
             return clazz.newInstance();
         }
-        catch (ReflectiveOperationException e) {
+        catch (ReflectiveOperationException | VerifyError e) {
```

Comment:
I think this is the wrong fix. We should have the compiler ensure the class is initialized, rather than catching `VerifyError` every place it could be used.